website/content/docs/configuration/target-aliases/create-target-alias.mdx

```diff
@@ -0,0 +1,539 @@
+      - **Target ID**: `tssh_lhH5pa425G`
+      - **Host ID**: `hst_7wGXkF8e0Q`
+   1. Click **Save**.
+   
+   </Tab>
+   <Tab heading="CLI" group="cli">
+   
+   1. Log in to Boundary.
+   1. Use the following command to create an alias for host `dev-041`:
+   
+      ```shell-session
+      $ boundary aliases create target \
+          -description 'Target alias for engineering.app-servers linux-dev dev-041' \
+          -destination-id tssh_lhH5pa425G \
+          -name dev-041 \
+          -scope-id global \
+          -value eng.app-servers.linux-dev.dev-041 \
+          -authorize-session-host-id hst_zlRwMMPKwp
+      ```
+
+      Example output:
+      
+      <CodeBlockConfig hideClipboard>
+      
+      ```shell-session
+      $ boundary aliases create target \
+             -description 'Target alias for engineering.app-servers linux-dev   dev-041' \
+             -destination-id tssh_lhH5pa425G \
+             -name dev-041 \
+             -scope-id global \
+             -value eng.app-servers.linux-dev.dev-041 \
+             -authorize-session-host-id hst_zlRwMMPKwp
+       
+       Alias information:
+         Created Time:        Thu, 14 Nov 2024 14:00:13 MST
+         Description:         Target alias for engineering.app-servers linux-dev dev-040
+         Destination ID:      tssh_lhH5pa425G
+         ID:                  alt_X5MRXRSi7t
+         Name:                dev-041
+         Type:                target
+         Updated Time:        Thu, 14 Nov 2024 14:00:13 MST
+         Value:               eng.app-servers.linux-dev.dev-041
+         Version:             1
+       
+         Scope:
+           ID:                global
+           Name:              global
+           Type:              global
+       
+         Authorized Actions:
+           no-op
+           read
+           update
+           delete
+       
+         Attributes:
+           authorize_session_arguments:
+           {
+           "host_id": "hst_zlRwMMPKwp"
+           }                                                                                                             
+      ```
+      
+      </CodeBlockConfig>
+   
+   </Tab>
+   </Tabs>
+
+You can now use the aliases to connect to the targets in different contexts.
+
+The Boundary Desktop Client app will list the `linux-dev-servers` target and its aliases under the **Aliases** column.
+
+When you click **Connect**, a list of the hosts available for the connection appears in the **Quick Connect** box.
+
+To establish a connection to any `linux-dev` host using the CLI, use the `eng.app-servers.linux-dev` alias:
+
+```shell-session
+$ boundary connect ssh eng.app-servers.linux-dev 
+```
+
+This will randomly select a host from the `linux-dev-servers` host set attached to the `linux-dev-servers` target.
+
+To establish a connection to a specific host, connect to its target alias instead:
+
+```shell-session
+$ boundary connect ssh eng.app-servers.linux-dev.dev-041
```

Comment:
Given the general advise that aliases should be dns-like, I would expect the aliases to be:

```
linux-dev.app-servers.eng
dev-040.linux-dev.app-servers.eng
dev-041.linux-dev.app-servers.eng
```

Reply:
great call out 👍

Reply:
I modified the aliases as suggested, and added a note about the alias pattern used in the example following `hostname.host-set.project.org`